src/algebra/category/Group.lean

```diff
@@ -91,6 +91,47 @@ induced_category.has_forget₂ (λ G : CommGroup, CommMon.of G)
 
 end CommGroup
 
+namespace AddCommGroup
+
+/-- Any element of an additive commutative group gives a unique morphism from `ℤ` sending
+`1` to that element. -/
+-- TODO allow other universe levels
+-- this will require writing a `ulift_instances.lean` file
+def as_hom {G : AddCommGroup.{0}} (g : G) : (AddCommGroup.of ℤ) ⟶ G :=
+{ to_fun := λ i : ℤ, i • g,
+  map_zero' := rfl,
+  map_add' := λ a b, gpow_add g a b }
+
+@[ext]
```

Comment:
Should this be given a higher priority? How do we make sure that the `ext` tactic picks this one up, instead of a more general ext lemma for homs.

Reply:
`ext` doesn't even accept priorities. The "mostly recently defined" `ext` rule wins, and you hope for the best.